ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/AddItemsToPortal/AddItemsToPortal.qml

```diff
@@ -17,14 +17,14 @@
 import QtQuick 2.6
 import QtQuick.Controls 2.2
 import Esri.Samples 1.0
-import Esri.ArcGISRuntime.Toolkit.Dialogs 100.10
+import Esri.ArcGISRuntime.Toolkit 100.9
```

Comment:
my mistake for not updating the issue, but we now have a beta_update10 branch which is based on 100.10 - https://github.com/Esri/arcgis-runtime-toolkit-qt/tree/beta_update10